### ISSUE TITLE
What about replacing "master" branch by "main" branch ?

### ISSUE THREAD

**Issue by srbarrios** [uyuni-project/uyuni#8620]
# Short description

The term master is out of favor in the computing world and beyond. Let's use the new standard and replace our trunk from "master" branch to "main" branch.

# Details

The computer industry's use of the terms [master and slave](https://www.theserverside.com/opinion/Master-slave-terminology-alternatives-you-can-use-right-now) caught everyone's attention in the summer of 2020. Amid the many protests and the growing social unrest, these harmful and antiquated terms were no longer considered appropriate.

Link: https://sfconservancy.org/news/2020/jun/23/gitbranchname/

"Both Conservancy and the Git project are aware that the initial branch name, 'master,' is offensive to some people and we empathize with those hurt by the use of that term," said the Software Freedom Conservancy.

# Steps

Git commands:
```
# Connect local repo with your remote repo
git remote add origin <remote_github_url>

# Pull the main source code
git pull origin main

# Move the source from master to main branch
git branch -m master main

# Add and commit your changes
git add .
git commit -m "master branch changed to main"

# Set upstream and push changes to remote
git push --set-upstream origin main
```

GitHub configuration:
- Change Default branch in project's settings

**Comment by szachovy** [comment]
I think that changing `Default branch` in project's settings would also be mandatory as a step.

**Comment by szachovy** [comment]
Not sure how our branch protection settings are configured; maybe some settings also need to be changed manually with other settings concerned.

**Comment by rjmateus** [comment]
there one missing pice, that is verify our workflow that can have the branch hardcoded

**Comment by mbussolotto** [comment]
also translation procedure would require some adjustment (e.g. branch name is `master-weblate` )

**Comment by srbarrios** [comment]
@admd @rjmateus  @uyuni-project/release-engineering should I close this idea?

**Comment by rjmateus** [comment]
From my point of view yes, it should be closed. Too much effort for almost no benefits. Unless we are forced to change it, I would not touch name and deal with all the impacts it will bring.

**Comment by admd** [comment]
I agree! Oscar, this will need more work so please close this for now. Or you can keep it around and we will revisit it after 5.2.
